app/src/components/Navbar.tsx

```diff
@@ -1,8 +1,14 @@
 import react, { type ReactElement } from "react";
 import { useNavigate } from "react-router-dom";
+import { LoginButton } from "./mock-login-button";
```

Comment:
note we are eventually gonna have to change these to actual frontend, maybe add a comment?